joplin.code-workspace

```diff
@@ -421,4 +421,4 @@
 			}
 		]
 	}
-}
\ No newline at end of file
+}
```

Comment:
remove

Reply:
Sorry, shouldn't be in the PR, thanks for fixing!

Reply:
Please remove the whitespace change

Reply:
Done